### ISSUE TITLE
Enable label positioning in Qslider

### ISSUE THREAD

**Issue by luckylooke** [quasarframework/quasar#9181]
**Is your feature request related to a problem? Please describe.**
Yes, we have design where label is under slider line and always visible... you have prop to make it always visible but there is not prop for making label to appear underneath the line.

**Describe the solution you'd like**
prop to control label position aka `label-position="< top | bottom | left | right >"` 

**Describe alternatives you've considered**
I think of some Qtooltip reuse here as it has already that [feature](https://quasar.dev/vue-components/tooltip#positioning) :shrug: 

Many thanks :pray:

**Comment by luckylooke** [comment]
![Screenshot from 2021-05-14 09-09-46](https://user-images.githubusercontent.com/3660030/118235149-a6b56f80-b494-11eb-8983-7db4aa339719.png)

As you can see, it even pretend from clicking on checkbox, could be `pointer-events: none;` :pray:

**Comment by smolinari** [comment]
Couldn't you just put some space between the slider and the checkbox?

Scott

**Comment by luckylooke** [comment]
No I can't, our designer guy would kill me :wink:  
I forced it for now:
```
  .q-slider {
    &.position-bottom,
    &.position-bottom.q-slider--editable:hover {
      ::v-deep {
        .q-slider__pin--h {
          transform: translateY(30px);
        }
        .q-slider__arrow--h {
          transform: translateY(0px) rotate(180deg);
        }
      }
    }
    ...
```
but I would appreciate the feature :pray:

**Comment by brent-williams** [comment]
Similar issue to @luckylooke, UI has vertical slider on far-right of UI, no obvious way to get the labels to appear to the left side of the slider so they're getting clipped.  Used similar CSS hack but results bit goofy, working built-in prop would rock.

```
.q-slider {
  ::v-deep {
    .q-slider__pin-text-container--v {
      transform: translate(-109px,-12px);
    }
    .q-slider__arrow--v {
      transform: translateX(-21px) rotate(180deg);
    }
  }
}
```

**Comment by rstoenescu** [comment]
Feature will be available in Quasar v2.4.0 (through new prop for both QSlider/QRange: switch-label-position).
A backport for Quasar v1 will be added shortly.
